Index: gst-plugins-good-1.9.90/gst/rtp/gstrtpvp9depay.c
===================================================================
--- gst-plugins-good-1.9.90.orig/gst/rtp/gstrtpvp9depay.c
+++ gst-plugins-good-1.9.90/gst/rtp/gstrtpvp9depay.c
@@ -174,8 +174,9 @@ gst_rtp_vp9_depay_process (GstRTPBaseDep
   }
 
   if (p_bit && f_bit) {
+    gint i;
     /* At least one P_DIFF|N, up to three times */
-    for (gint i = 0; i < 3; i++) {
+    for (i = 0; i < 3; i++) {
       guint p_diff, n_bit;
 
       if (G_UNLIKELY (size < hdrsize + 1))
